### ISSUE TITLE
[1.12.2] Blocks with textures utilizing CTM's .mcmeta features broken by OSV

### ISSUE THREAD

**Issue by SuperiorPineapple** [PersonTheCat/ore_stone_variants#130]
This is strictly an appearance-related issue.
When any block, vanilla or modded, is made to use any of CTM's features, its graphics cease working as intended or break completely when said block is converted into an ore variant. Emissive textures cease working, graphics load incorrectly upon triggering connections, and "random" type CTM textures load whole instead of the intended random grid segment.

![obraz](https://user-images.githubusercontent.com/88989048/129493931-1064ff01-965d-4ed5-bd36-30899a084d3a.png)
Example above is vanilla ores and BoP's tanzanite to the right, with CTM's emissive textures applied. To the left are regular stone variants of the same ores, provided by OSV, with CTM's emissive textures applied in the identical way. There should be no difference, but due to the incompatbility, the intended glow is not present at all. While this in particular is not a major loss, this compatibility issue becomes increasingly gross to look at when other CTM features are in play.

![obraz](https://user-images.githubusercontent.com/88989048/129494443-a83ca3d0-4a70-42c9-b53c-e4e6d1578e4c.png)
In this example, the bottom texture of the sandstone block uses a 4x4 tile grid for CTM randomization, but being used for an ore variant causes this texture to break and load whole.

![obraz](https://user-images.githubusercontent.com/88989048/129494499-a7546ff5-26d6-4854-aed2-f5ce02ad3a98.png)
Finally, an example of regualr CTM using sandstone. The original block works as intended, connecting to itself and loading the specified CTM texture. The coal ore variant is recognized and regular sandstone connects to it - but it refuses to connect itself to surrounding blocks and other instances of itself, despite using the exact same texture which uses the same .mcmeta file.

**Comment by WenXin20** [comment]
To add to this with 1.16.5, CTM will apply but overwrites the ore overlay texture.

**Comment by Stormalisk** [comment]
Going to put this in here as I don't know if would be proper to make a new topic for it.
With CTM 1.1.2.6 and OSV 7.0 causes a crash on load
[crash-2021-12-15_00.37.32-client.txt](https://github.com/PersonTheCat/OreStoneVariants/files/7716724/crash-2021-12-15_00.37.32-client.txt)

**Comment by PersonTheCat** [comment]
Yeah, looks like they load resource packs at a super early time / before any OSV code is even available. I can probably just have it ignore this issue. Will do a small bug fix update in the next couple of weeks.

**Comment by PersonTheCat** [comment]
Hmm, looks like CTM passed a null value in for the method used to register resource packs. I may still fix this because it will be a really small change, but if I'm correct, they really need to fix it and not me. I'll investigate the issue more in the morning.

**Comment by PersonTheCat** [comment]
Hey, @Stormalisk. I tested OSV with CTM and did not experience a crash. I assume this issue is actually coming from Chisel. I see that Chisel is in alpha and not on CurseForge, so I can't test it. Try again without Chisel and see if the issue goes away.

**Comment by Stormalisk** [comment]
Okay so it was Chisel, and also Chipped mod?
I ran the previous tests in a clean instance with CTM and Chisel. I first encountered it on my modpack which does not have Chisel, but the Chipped mod (which is similar).

I'll attach that report here too, I didn't post it before because I figured a cleaner test environment would help more and I happened to blame CTM first
[crash-2021-12-15_05.27.57-client.txt](https://github.com/PersonTheCat/OreStoneVariants/files/7732193/crash-2021-12-15_05.27.57-client.txt)
.
